src/styles/_variables.scss

```diff
@@ -0,0 +1,52 @@
+@import url(https://fonts.googleapis.com/css2?family=Poiret+One&display=swap);
+@import url(https://fonts.googleapis.com/css2?family=Comfortaa:wght@300;400;500;600;700&display=swap);
+@import url(https://fonts.googleapis.com/css2?family=Comfortaa:wght@300;400;500;600;700&family=Oswald:wght@200;300;400;500&display=swap);
+//font families
```

Comment:
Wow, lots of CSS here!  I'm wondering if we are utilizing all of these variables currently, or if these are being stored here for future reference.

Reply:
We are not using all of them, probably half and I have them right now for future reference.